scripts/create-component.js

```diff
@@ -41,10 +29,9 @@ const updateIndexTSFile = component => {
     encoding: 'utf-8',
   });
 
-  const exportComponent = `export { ${component} } from './${component}';\n`;
```

Comment:
I believe this line is necessary. I believe it needs a correction because if you observe it is importing a component that is not used in the export below, can you see that?

Reply:
i agree